tests/features/ecas-register.feature

```diff
@@ -1,12 +1,22 @@
-@javascript @ecas-login
+@api @ecas-login
 Feature: Register through OE Authentication
   In order to be able to have new users
   As an anonymous user of the system
   I need to be able to go to the registration URL
 
+  @DrupalLogin @BackupAuthConfigs @RebuildRouter
```

Comment:
OK I see here that there is this `@BackupAuthConfigs` tag which presumably takes care of restoring the configuration.

Can we maybe extend this hook to rebuild the router?

I don't really like this approach of changing configuration through the UI, and then relying on tags to magically restore the values, but if this is the existing convention followed by the project then it is better to follow it.

But let's try not to end up with more tags than necessary :)